### PR TITLE
Resolution switching examples

### PR DESCRIPTION
**Pull request by nwtn**

### CODE REVIEW

File: index.src.html [hunk 1]
```diff
@@ -312,9 +312,25 @@ <h2 class="no-num no-toc">Table of contents</h2>
...
-    <li>Device pixel ratio based resolution switching - Different screen densities require different image resolutions in order to display
-    the same image dimensions in a crisp, artifact-free way</li>
-    <li>Viewport dimensions based resolution switching - Different viewport sizes result in different image dimensions </li>
+    <li>
+    	Device pixel ratio based resolution switching - Different screen densities require different image resolutions in order to display
+    the same image dimensions in a crisp, artifact-free way
+	<figure>
+	  <div><img src="images/dpr-camera-man.jpg" width="1200" height="320" style="max-width:1200px" class="stretchy" alt="an image displayed at 1x DPR, the same resolution stretched to display at 2x DPR, and with double the resolution at 2x DPR" /></div>
+	  <figcaption>
+	    <p>On a device with a device pixel ration greater than 1, higher image resolutions are required to eliminate artifacts.</p>
```
Comment (marcoscaceres) [inline index.src.html]: typo: ration

File: index.src.html [hunk 2]
```diff
@@ -312,9 +312,25 @@ <h2 class="no-num no-toc">Table of contents</h2>
...
+	  </figcaption>
+	</figure>
+	</li>
+    <li>
+    	Viewport dimensions based resolution switching - Different viewport sizes result in different image dimensions
+		<figure>
+		  <div><img src="images/viewport-camera-man.jpg" width="960" height="320" style="max-width:960px" class="stretchy" alt="the same photograph at two resolutions, but displayed in equal-width viewports, will look identical" /></div>
+		  <figcaption>
+		    <p>To optimize performance, images that match the viewport size as closely as possible should be displayed.</p>
```
Comment (marcoscaceres) [inline index.src.html]: Please avoid use of passive voice - change to active voice.

File: index.src.html [hunk 3]
```diff
@@ -312,9 +312,25 @@ <h2 class="no-num no-toc">Table of contents</h2>
...
+	    <p>On a device with a device pixel ratio greater than 1, higher image resolutions are required to eliminate artifacts.</p>
+	  </figcaption>
+	</figure>
+	</li>
+    <li>
+    	Viewport dimensions based resolution switching - Different viewport sizes result in different image dimensions
+		<figure>
+		  <div><img src="images/viewport-camera-man.jpg" width="960" height="320" style="max-width:960px" class="stretchy" alt="the same photograph at two resolutions, but displayed in equal-width viewports, will look identical" /></div>
+		  <figcaption>
+		    <p>To optimize performance, authors can use images that more closely match the viewport size.</p>
```
Comment (yoavweiss) [inline index.src.html:330]: Maybe something like "This illustration shows how on a single viewport size, the larger image has no positive impact on the user perceived image quality, while having a negative impact on the page's performance, by downloading excessive image data. 
To optimize performance, images that match the viewport size as closely as possible should be downloaded"

**Comment by yoavweiss** [comment]
Merging so that @marcoscaceres can move the spec to ReSpec. I'll address the few comments I had in a separate PR.

**Pull request merged**